src/main.ts

```diff
@@ -165,6 +169,45 @@ const command: Command = {
 		else {
 			return compile(config(configArgs), options);
 		}
+	},
+	eject(helper: Helper) {
+		const ejectOutput: EjectOutput = {
+			npm: {
+				devDependencies: {
```

Comment:
Is there any way this data could be pulled from the `dependencies` listed `package.json` (with the exception of `@dojo/cli-build-webpack`)? That way, this list wouldn't have to be maintained in two different places.